latest-langs

```diff
@@ -89,6 +90,7 @@ for %langs{ @langs || * }:p.sort: *.key.fc -> (:key($name), :value(%lang)) {
         state $ua  = HTTP::Tiny.new :throw-exceptions;
         my    $res = $ua.get("https://$_")<content>.decode;
 
+        when / 'aplwiki'     / { $res ~~ / ' release' .+? <(<ver>)> / }
```

Comment:
Dyalog APL, and APL derivatives BQN, Jelly, and Uiua are all making use of this new domain. That's a lie, Uiua not yet, but that's about to change hopefully. I'd say we're at a point where it can't hurt squeezing that domain in.

Reply:
Even J and K.

![image](https://github.com/user-attachments/assets/1a042f01-c26f-465b-8267-319375b2131a)

Reply:
Lil has been, but is no more. Nial's an unaddressed issue, for now.